### PR TITLE
Improve example READMEs

### PR DESCRIPTION
**Pull request by tobloef**
This PR improves the existing READMEs for the three examples by adding a few missing steps and tweaking the text to be more readable.

### CODE REVIEW

File: examples/core-example-advanced/README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ An advanced example project for `@tldraw/core`.
 
 To start this project:
 
-- run `yarn start:core` from the repository's root directory
-- open `localhost://5421` in your browser
-- OR run `yarn dev` from the `examples/core-example-advanced` directory
+1. Run `yarn` from the repository's root directory
+2. Run `yarn start:core` rom the repository's root directory
```
Comment (alex-kim-dev) [inline examples/core-example-advanced/README.md:12]: A little typo in '*f*rom', same in the `tldraw-example` package

Comment (tobloef) [reply examples/core-example-advanced/README.md]: fixed, good catch

**Review by alex-kim-dev — APPROVED** [review]
Seems good to me

**Comment by steveruizok** [comment]
Thanks!

**Pull request merged**